hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java

```diff
@@ -830,9 +830,15 @@ private String getQueueClassPrefix() {
   public synchronized void refreshCallQueue(Configuration conf) {
     // Create the next queue
     String prefix = getQueueClassPrefix();
-    this.maxQueueSize = handlerCount * conf.getInt(
-        CommonConfigurationKeys.IPC_SERVER_HANDLER_QUEUE_SIZE_KEY,
-        CommonConfigurationKeys.IPC_SERVER_HANDLER_QUEUE_SIZE_DEFAULT);
+    String handlerQueueSizeStr = conf.get(prefix + "." +
```

Comment:
Would it be better to use ```conf.getInt``` here?

Reply:
Thanks @tomscut  for the comment.
This is my neglect, I will update it later.